### PR TITLE
test(logging): add timedict test

### PR DESCRIPTION
**Pull request by Yongxuanzhang**

### CODE REVIEW

File: tests/unit/logging/test_logging_profile.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import time
+
+
+def test_logging_profile_timedict():
+    td = TimeDict()
+    td("timer")
+    with td:
+        time.sleep(2)
+
+    assert int(td.accum_time["timer"]) == 2
```
Comment (JoanFM) [inline tests/unit/logging/test_logging_profile.py:11]: can u please use `single quotes`? Otherwise, great job!

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: tests/unit/logging/test_logging_profile.py (hunk 1 above)
Comment (Yongxuanzhang) [reply tests/unit/logging/test_logging_profile.py]: Sure! Thanks!

**Review by JoanFM — APPROVED** [review]
LGTM!

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**